config.py

```diff
@@ -0,0 +1,40 @@
+import dotenv
+import os
+import urllib.parse as urlparse
+
+if os.path.isfile(".env"):
```

Comment:
You could just do `dotenv.load_dotenv()` so it finds the env files on its own

Reply:
@takos22 what happens if it doesnt file any file?

Reply:
when we use it in kubernetes we wont have a .env file we would just straight up have those in env vars

Reply:
> @takos22 what happens if it doesnt file any file?

It wouldn't load anything and might raise a KeyError if you try to access them

Reply:
But in our case it doesn't change anything